package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
Does this problem still show up? I noticed that the file transfer plugin bumped their dependency version of the file plugin to ^4.0.0. https://github.com/apache/cordova-plugin-file-transfer/commit/3540af439ed0a91885ff2cbf5e0a3d42c7fd05ed#diff-53f390d375398624afe1cfe1125f42bfR32

As part of a seperate discussion, I think we should change all of our usage of `>=` to `^`.